### ISSUE TITLE
Design Module to persist credentials properly

### ISSUE THREAD

**Issue by ChrisCates** [lynndylanhurley/redux-auth#66]
I have patched the code with a npm module `redux-auth-patch`.
It appears there are some issues with persisting authHeaders in the cookies.

the `isApiRequest()` will break if you do not clear cookies upon new entry.

I'd also suggest a more procedural type approach to the `session-storage.js` file. It was quite confusing tracing the issue and then identifying that the issue was in fact in `fetch.js` not `session-storage.js`.

**Comment by lynndylanhurley** [comment]
PRs welcome.

**Comment by lynndylanhurley** [comment]
I agree with you about the code organization stuff. Working on cleaning things up with the next release (`v0.0.6`)
